examples/plot_classification_example.py

```diff
@@ -0,0 +1,244 @@
+            linewidth=1,
+            marker="*",
+            s=10,
+            legend=False,
+            dodge=0.5,
+            orient="h",
+        )
+        sns.boxplot(
+            data=df_method,
+            x="importance",
+            y="feature",
+            hue="model",
+            ax=ax,
+            legend=legend,
+            orient="h",
+        )
+        ax.set_title(method, fontweight="bold")
+        ax.axvline(0, color="k", linestyle="--")
+        ax.set_xlabel("Importance")
+        ax.set_ylabel("")
+        for i in range(len(df_method["feature"].unique())):
+            if i % 2 == 0:
+                ax.axhspan(i - 0.5, i + 0.5, color="gray", alpha=0.33)
+
+    ax = axes[-1]
+    handles, labels = ax.get_legend_handles_labels()
+    handles.append(
+        plt.Line2D(
+            [0],
+            [0],
+            marker="*",
+            color="w",
+            markerfacecolor="w",
+            markeredgecolor="k",
+            markersize=12,
+            label=f"pval < {threshold}",
+            markeredgewidth=1.5,
+        )
+    )
+    ax.legend(
+        handles=handles,
+        labels=labels + [f"pval < {threshold}"],
+        loc="center left",
+        bbox_to_anchor=(1, 0.5),
+    )
+    plt.tight_layout()
+    plt.show()
+
+
+plot_results(df, df_pval)
+
+
+####################################################################################
+# The boxplot shows the importance of each feature, with colors indicating the
+# classifier used. A star marks the features that have a p-value (computed with a
+# t-test) below 0.05. As expected, the spurious feature is not selected by LOCO and CPI,
+# but is selected by Permutation Importance.
+
+
+#########################################################################
+# Measuring the importance of groups of features
+# -----------------------------------------------------------------------
+# In the example above, CPI and LOCO did not select some features. This is because they
+# measure conditional importance, which is the additional independent information a
+# feature provides knowing all the other features. When features are highly correlated,
+# this additional information decreases, resulting in lower importance rankings. To
+# mitigate this issue, we can group correlated features together and measure the
+# importance of these feature groups. For instance, we can group 'sepal width' with
+# 'sepal length' and 'petal length' with 'petal width' and the spurious feature.
+groups = {"sepal features": [0, 1], "petal features": [2, 3, 4]}
+out_list = Parallel(n_jobs=5)(
+    delayed(run_one_fold)(
+        X, y, model, train_index, test_index, vim_name=vim_name, groups=groups
+    )
+    for train_index, test_index in cv.split(X)
+    for model in models
+    for vim_name in ["CPI", "LOCO", "PermutationImportance"]
+)
+
+df_grouped = pd.concat(out_list)
+df_pval = compute_pval(df_grouped, threshold=threshold)
+plot_results(df_grouped, df_pval)
```

Comment:
Why LOCO doesn't select the 2 groups at then end?

Reply:
My explanation would be that LOCO has less statistical power because of the larger variance it introduces when refitting sub-models. When the sample size is small (Iris has 150 samples ), this leads larger p-values.